rework RTT subsection (#516)
quicwg/multipath: draft-ietf-quic-multipath.md

## Computing Path RTT {#compute-rtt}

> PATH_ACK frames indicates which path the acknowledged packets were sent on,

Reviewer: ```suggestion
PATH_ACK frames indicate which path the acknowledged packets were sent on,
```

but they may be received through any open path. If successive acknowledgments are received
on different paths, the measured RTT samples can fluctuate widely,

> which could result in poor performance depending e.g. on the usd connection control.

Reviewer: Used congestion control?

Congestion control state as defined in {{QUIC-RECOVERY}} is kept
per Path ID. However, depending on which path acknowledgements are

> sent, the actually RTT of a path cannot be calculated or may not be

Reviewer: ```suggestion
sent, the actual RTT of a path cannot be calculated or may not be
```

Satellite   | 350ms  | 600ms
{: #fig-example-ack-delay title="Example of ACK delays using multiple paths"}

The computed values reflect both the state of the network path and the
scheduling decisions of the acknowledgement sender. If we
assume that the PATH_ACK will be sent over the terrestrial
link, because that provides the best response time, the
computed RTT value for the satellite path will be about 350ms. This is
lower than the maximum of 600ms that would be measured if the PATH_ACK came over
the satellite channel, but it is still the right value for computing
for example the PTO timeout: if a PATH_ACK is not received after more
than 350ms, either the data packet or its PATH_ACK were probably lost.

> The simplest implementation is to used the acknowledgment delay

Reviewer: ```suggestion
The simplest implementation is to use the acknowledgment delay
```

to compute smoothed_rtt and rttvar per
{{Section 5.3 of QUIC-RECOVERY}} regardless of the path through which PATH_ACK frames are
received. This approach will provide good results
as long as acknowledgements are sent consistently over the same path.
If the acknowledgement sender

> revisits its sending preferences, the set of paths can change. However, this is not very

Author: yes, do you want to propose a different (short) wording?

> revisits its sending preferences, the set of paths can change. However, this is not very

Author: @qdeconinck any wording proposal? Or is this okay as it is?

> If the acknowledgement sender
> revisits its sending preferences, the set of paths can change. However, this is not very

Reviewer 2: Maybe a suggestion, but if this does not work (as it is longer) we can stick with the current text
```suggestion
However, endpoints may at some point revisit sending preferences, leading to
other paths being used to send acknowledgements. However, this is not very
```

Author: ```suggestion
If at any time the sender revisits its sending preferences,
this can also change the paths that are used to send acknowledgements.
However, this is not very
```

Author: @qdeconinck Is this okay?

Reviewer 2: @mirjak Yes, let's take this last version.

different from route changes on a single path.
The RTT, RTT variance and PTO estimates will rapidly converge to
reflect the new conditions.
There is however one exception: the minimum RTT. But this is also

> a known challenge when route changes ocurrs on a single path.  

Author: ```suggestion
a known challenge when route changes ocurrs on a single path.
```
